### ISSUE TITLE
Feature Request: Improve Lighthouse SEO Score

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#102]
### Describe the feature

The site currently has a score of 77 for SEO when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110366797-afc23480-803e-11eb-8f5b-429df34c41c4.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the SEO of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for SEO is increased to at least 90.
